src/Tuura/Concept/STG/Translation.hs

```diff
@@ -94,3 +85,10 @@ initVal s ls = sum (map (\x -> if (fst x == s) then fromEnum (snd x) else 0) ls)
 
 readArc :: String -> String -> [String]
 readArc f t = [f ++ " " ++ t, t ++ " " ++ f]
+
+genInvStrs :: (Eq a, Show a) => [[Transition a]] -> [String]
+genInvStrs invs
+        | invs      == [] = []
+        | otherwise = info ++ map (\i -> unwords (map show i)) invs
+    where
+        info = ["Signal transition sets which are invariant:"]
```

Comment:
Would this not get lost in the parsing of the `.g` file though? For example, in Workcraft, do we not want this information to be output to the user?